glibc_wrap/glibc_wrap.c

```diff
@@ -0,0 +1,33 @@
+// Derived from the version used in Quartus by Devin Sundaram and Jeff Da Silva:
+// (https://github.com/intel-innersource/applications.fpga.quartus-prime.glibc-wrap).
```

Comment:
It would be good to try avoid references to internal links since the runtime repo is a public one, and this link will appear as invalid address to users.